app/scripts/modules/core/src/pipeline/executions/executionGroup/ExecutionGroup.tsx

```diff
@@ -241,7 +240,7 @@ export class ExecutionGroup extends React.PureComponent<IExecutionGroupProps, IE
             execution={execution}
             pipelineConfig={pipelineConfig}
             application={this.props.application}
-            onRerun={pipelineConfig && isConfigurable ? this.rerunExecutionClicked : undefined}
```

Comment:
Just trying to understand why we can remove the `isConfigurable` check -- it looks like `isConfigurable` checked 1) that the feature flag is on, and 2) if the config has the `v2` schema. Why do we no longer need to check 2?

Reply:
Yeah this is tricky. Previously, the isConfigurable const was checking that if there was a pipeline config that it would not be a v2 kind if the flag was off. If the flag was on, then it would let the pipeline config pass the check regardless of type. Going forward all pipeline configs can be treated as configurable.